web3/app.html

```diff
@@ -0,0 +1,265 @@
+<script src="https://cdn.jsdelivr.net/npm/web3@latest/dist/web3.min.js"></script>
+
+<script>
+console.log("Script started")
+
+window.addEventListener('load', function () {
+  console.log("Loading window")
+
+  const walletButton = document.querySelector("#walletButton")
+  const balanceLabel = document.querySelector("#balanceLabel")
+  const priceLabel = document.querySelector("#priceLabel")
+  const pizzasLabel = document.querySelector("#pizzasLabel")
+  const buyButton = document.querySelector("#buyButton")
+
+  // Placeholders
+  balanceLabel.innerHTML = balance + " Ξ"
+  priceLabel.innerHTML = price + " ₿"
```

Comment:
I think this is super confusing for users. Just show the price in ETH Ξ because that is what they are paying in. And display BTC price next to it or under

Reply:
Agreed, just left it like this since I’m not sure a decision has been made in regards to how we should show the price because of the 10k btc narrative - do you have confirmation from Sam?

Reply:
I think i should move the BTC price out of it. and sommewhere else maybe? is there a js embed i can throw above it or something? I think pizza counter is better.